bootloader/bootloader.go

```diff
@@ -192,6 +192,12 @@ type TrustedAssetsBootloader interface {
 	// the bootloader's rootdir that are measured in the boot process in the
 	// order of loading during the boot.
 	TrustedAssets() ([]string, error)
+
+	// RecoveryBootChain returns the chain of recovery mode boot files.
+	RecoveryBootChain(kernelPath string) ([]BootFile, error)
+
+	// RecoveryBootChain returns the chain of run mode boot files.
```

Comment:
// BootChain returns the load chain for run mode.
// It should be called on a RoleRecovery bootloader
// passing the RoleRunMode bootloader.